editor/src/clj/editor/console.clj

```diff
@@ -298,16 +318,17 @@
 
 (defn- init-console-filter! [filter-console-button prefs]
   (let [filters (prefs/get prefs console-filters-prefs-key)
-        state (atom {:open false :text "" :filters filters})]
-    (set-filters! filters)
+        filtering (prefs/get prefs console-filtering-key)
+        state (atom {:open false :enabled filtering :text "" :filters filters})]
+    (set-filters! (if filtering filters []))
     (fx/mount-renderer
       state
       (fx/create-renderer
         :error-handler error-reporting/report-exception!
         :opts {:fx.opt/map-event-handler #(handle-filter-event! state prefs %)}
         :middleware (comp
                       fxui/wrap-dedupe-desc
-                      (fx/wrap-map-desc #(filter-console-view filter-console-button %)))))))
+                      (fx/wrap-map-desc #(filter-console-view filter-console-button prefs %)))))))
```

Comment:
If we update the `filtering` preference in `handle-filter-event!` like we currently do for `filters`, then passing the prefs to view will be unnecessary.